CHANGELOG.md

```diff
@@ -1,3 +1,51 @@
+# 93.1.0
+
+This minor release deprecates the `buttonFont` API of `MDCAlertController`.
```

Comment:
Is this a hotfix release? If so please indicate as such.

Reply:
Done.